fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -171,13 +172,16 @@ class BrowserToolbarView(
                     isPrivate = customTabSession.content.private,
                 )
             } else {
+                val useNewDesign = IncompleteRedesignToolbarFeature(context.settings()).isEnabled
```

Comment:
I find names like `useNewDesign` could be confusing down the road for people looking at the code and not having a lot of context on the redesign. We could hint the nature of the new redesign by using a name like 'isNavBarEnabled' – so a reader of the code even not having the context about the redesign project would understand what's happening

Reply:
That's a good idea, I'll rename the param